src/com/lftechnology/java/training/sanish/stemleaf/StemLeaf.java

```diff
@@ -0,0 +1,43 @@
+package com.lftechnology.java.training.sanish.stemleaf;
+
+public class StemLeaf {
+	private int stem;
+	private int leaf;
+	private int count;
+
+	private StemLeaf(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)